driver/src/driver/stablex_driver.rs

```diff
@@ -47,6 +47,23 @@ impl<'a> StableXDriver<'a> {
             return Ok(false);
         }
 
+        #[cfg(not(test))]
```

Comment:
nit: can we move this below `auction_processing_started`? I feel like we could still measure when we "started" the processing.